### ISSUE TITLE
NVDA cannot access the address book of Thunderbird 

### ISSUE THREAD

**Issue by lstar4ever** [nvaccess/nvda#10743]
#6045 # Steps to reproduce:
1. Open Thunderbird
2. open the e
address book by ctrl shift b or from the tools menu
3. tab onto the adress book window and found that nvda cannot review the content, only speak unknown.
4. tab several times nvda can only identify the quick search box, but nvda cannot review the search result
 
### Actual behavior:
Tab into the address book window nvda only speak unknow.

### Expected behavior:
when tab into the adress window, user should be able to use up or down arrow to review different stored adress book.

Within each adress book, user should be able to use up or down arrow to review the stored contact, and nvda should speak the name, email, phone number etc.

#### NVDA installed/portable/running from source:
install

#### NVDA version:
2019.2.1
#### Windows version:
10 1909
#### Name and version of other software in use when reproducing the issue:
Thunderbird68.4.2
#### Does the issue still occur after restarting your PC?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no

**Comment by bdorer** [comment]
Hi, I'm not able to reproduce this. Please start NVDA without addons.

Thanks

Am 01.02.2020 um 11:21 schrieb lstar4ever:

**Comment by lstar4ever** [comment]
Thanks.
I tried nvda -rm --disableaddons, and the problem is still here.

I just tried using Thunderbird 60.9.1 and nvda can access the adress book without problem.

**Comment by k-kolev1985** [comment]
I think this is an issue in Thunderbird 68.x.x itself. It affects not only the address book, but the search results list in the advanced search (Control+Shift+F) as well. But it does not occur all the times - only sometimes and I think a restart of Thunderbird fixes it. Now I cannot reproduce it, but when it happens, if I don't forget, I'll get the NVDA log for it.

P.S.: If I remember correctly, when I was able to reproduce the bug, I tested it with Narrator and JAWS too and the problem was the same, so that is why I think it is not an NVDA issue but one in Thunderbird itself.

**Comment by Adriani90** [comment]
Yeah it seems it is a bug in Thunderbird, I could reproduce it from time to time but restarted Thunderbird and everything was ok again. Also don't forget to re-register the COM interface from time to time by running the COM registration fixing tool in NVDA menu under tools.
Closing as this needs an external fix and is invalid for NVDA.
Please contact the developers from Thunderbird regarding this issue. Thanks.

**Comment by k-kolev1985** [comment]
I've managed to reproduce the issue. For me it occurs only for the search results list in the advanced search (Control+Shift+F), but not for the address book (Control+Shift+B). And I didn't have to even restart Thunderbird - I switched to it from the Alt+TAB list, opened the advanced search, performed an advanced search and it occurred.

### Test environment:
- Operating system: Windows 10 Pro version 1909 (build 18363.628), 64-bit, in Bulgarian with all locale settings set to "Bulgarian".
- NVDA version: alpha-19604,7773e8ff, in bulgarian.
- Mozilla Thunderbird version: 68.4.2, 32-bit, in bulgarian, with the following enabled/active extensions:
  - ImportExportTools NG v4.0.4.
  - Manually sort folders v2.0.2.
  - Quicktext v2.12.
  - Xpunge v1.0.0.
- Processor: Intel Core i5-9400F at 2.90GHz.
- RAM Memory: 16.00GB.
- Graphics: MSI GeForce GTX-1050TI Gaming X 4G, 4096MB dedicated memory, desktop resolution set to 1920x1080.
- Sound Card: Realtek ALC892 at Intel Cannon Point PCH.

I'm attaching the NVDA log file with the issue reproduced. The testing was done with NVDA add-ons disabled.

[NVDA_Log_2020-02-01_#10743.log](https://github.com/nvaccess/nvda/files/4142507/NVDA_Log_2020-02-01_.10743.log)
